frontend/src/components/file-explorer/FileExplorer.tsx

```diff
@@ -111,7 +111,7 @@ function FileExplorer() {
       const fileList = await listFiles("/");
       setFiles(fileList);
       if (fileList.length === 0) {
-        toast.info(t(I18nKey.EXPLORER$EMPTY_WORKSPACE_MESSAGE));
+        toast.info(t(I18nKey.EXPLORER$EMPTY_WORKSPACE_MESSAGE), true);
       }
```

Comment:
Instead of all the changes, I'd actually be for removing here lines 113-115 completely and get rid of this toast completely.

Reply:
Why the toast was added?